config/7.6.0/common.yaml

```diff
@@ -837,14 +837,75 @@ others:
         rules:
           - apiGroups:
               - ""
-              - app.kiegroup.org
+            resources:
+              - configmaps
+              - pods
+              - services
+              - serviceaccounts
+              - persistentvolumeclaims
+              - secrets
```

Comment:
my main question here is for @errantepiphany @rhtevan  ... does PAM need all of this access? I've put this here instead of the wildcard access we are currently declaring, but it still seems excessive. 

Specifically, what does PAM need writes to beyond view/edit rights for ConfigMaps?  Are there any other openshift/k8s objects that PAM interacts with at runtime?

Reply:
AFAIK, PAM needs to be able to manipulate ConfigMap, watch and trigger rollout for DeploymentConfig, but not interacts with Secrets. PAM image scripts interact with KubeAPI server for querying.

Reply:
so i can remove the other resource types from this Role? Just leave ConfigMaps and DeploymentConfigs?

@rhtevan @errantepiphany  ?

Reply:
I would say at least keep 'serviceaccounts'. @errantepiphany  anything else needed from script perspective?